keps/sig-storage/1710-selinux-relabeling/README.md

```diff
@@ -403,7 +403,7 @@ _This section must be completed when targeting alpha to a release._
 
 * **How can this feature be enabled / disabled in a live cluster?**
   - [X] Feature gate (also fill in values in `kep.yaml`)
-    - Feature gate name: SELinuxRelabelPolicy
+    - Feature gate name: `SELinuxMountReadWriteOncePod`
     - Components depending on the feature gate: apiserver (API validation only), kubelet
   - [ ] Other
     - Describe the mechanism: 
```

Comment:
I didn't notice this last time.  You're missing an answer for this section. i believe you need need a drain and restart of the kubelet to change this setting.  Please describe that.

Reply:
> I didn't notice this last time. You're missing an answer for this section. i believe you need need a drain and restart of the kubelet to change this setting. Please describe that.

I see the answer a few questions down.